ci/test-install/storage_install_test.cc

```diff
@@ -47,10 +47,11 @@ int main(int argc, char* argv[]) try {
   while (std::getline(stream, line, '\n')) {
     ++count;
   }
-  client
-      .DeleteObject(bucket_name, object_name,
-                    gcs::Generation(meta.generation()))
-      .value();
+  google::cloud::Status status = client.DeleteObject(
+      bucket_name, object_name, gcs::Generation(meta.generation()));
+  if (!status.ok()) {
```

Comment:
Not necessarily in the scope of this PR, but I see you adding `.ok()` when I believe we're pushing towards `if (!status)`. Would it save future effort to remove all `.ok()`'s?

Reply:
Unless `Status` can't do that then ignore this comment.

Reply:
Right ... `Status` can't do that, and I believe the plan of record is that it never will.